### DISCUSSION TITLE
installation via homebrew is janky

### DISCUSSION THREAD

**Discussion by s3r3n3ty** [nicotine-plus/nicotine-plus#3084] [Q&A] [answered]
Installation via homebrew which causes two bits of ux jankieness which are not present by just using the .dmg installer from the website directly.
1.) there is no application shortcut made, to open one has to go into ones homebrew dir and directly launch the exec
2.) instead of the expected dock icon and proper name in the top bar, the latter shows simply 'Python' and the dock icon is some default python thingy instead of the nicotine icon

I didn't want to open a bug bc this problem only exists when installing via homebrew not when installing manually from the provided .dmg and maybe I'm just being dumb but still wanted to bring it up.
Is this expected behaviour or should I open a bug?

<img width="1512" alt="Screenshot 2024-08-18 at 10 13 47" src="https://github.com/user-attachments/assets/c5a38754-3989-4dee-ad97-a8771ece5ad1">

**Comment by mathiascode** [comment, marked as answer]
Yes, it's known. Publishing our .dmg on Homebrew Cask would be the better option, but we can't do that since the .dmg isn't signed.
